src/openrct2/ride/Vehicle.cpp

```diff
@@ -2671,7 +2671,7 @@ static bool ride_station_can_depart_synchronised(const Ride& ride, StationIndex
      */
 
     int32_t direction = tileElement->GetDirectionWithOffset(1);
-    int32_t maxCheckDistance = RIDE_ADJACENCY_CHECK_DISTANCE;
+    int32_t maxCheckDistance = kRideAdjacencyCheckDistance;
     int32_t spaceBetween = maxCheckDistance;
```

Comment:
I wonder if we shouldn't have `maxCheckDistance` and `spaceBetween` updated to `uint8_t` too. They are never compared to anything signed.

Btw `maxCheckDistance` in here can also be made `constexpr`.

Reply:
Seems a good idea to me, also making `maxCheckDistance` a `constexpr` fine by me. I will make the change and push.